include/boost/geometry/algorithms/detail/overlay/enrich_intersection_points.hpp

```diff
@@ -534,9 +540,15 @@ inline void enrich_intersection_points(Turns& turns,
     }
 
 #ifdef BOOST_GEOMETRY_DEBUG_ENRICH
-    //detail::overlay::check_graph(turns, for_operation);
+    constexpr bool do_check_graph = true;
+#else
+    constexpr bool do_check_graph = false;
 #endif
 
+    if BOOST_GEOMETRY_CONSTEXPR (do_check_graph)
+    {
+        detail::overlay::check_graph(turns, target_operation);
+    }
```

Comment:
(Slightly) related change which enforces compilation of `check_graph` (it was broken in 2 ways,
one by my PR, the other because `for_operation` is renamed somewhen).

@vissarion is this OK?

Reply:
yes, thanks.